test/tests.bats

```diff
@@ -670,7 +672,7 @@ EOF
 @test "checking accounts: listmailuser (quotas enabled)" {
   run docker exec mail /bin/sh -c "sed -i '/ENABLE_QUOTAS=0/d' /etc/dms-settings; listmailuser | head -n 1"
   assert_success
-  assert_output '* user1@localhost.localdomain ( 12K / ~ ) [0%]'
+  assert_output '* user1@localhost.localdomain ( 13K / ~ ) [0%]'
```

Comment:
```suggestion
  assert_output '* user1@localhost.localdomain ( 14K / ~ ) [0%]'
```

Based on new test failure since rebasing to current master, for some reason the mailbox size in this test increased by 1KiB:

```
not ok 318 checking accounts: listmailuser (quotas enabled) in 607ms
# (from function `assert_output' in file test/test_helper/bats-assert/src/assert.bash, line 239,
#  in test file test/tests.bats, line 675)
#   `assert_output '* user1@localhost.localdomain ( 13K / ~ ) [0%]'' failed
# 
# -- output differs --
# expected : * user1@localhost.localdomain ( 13K / ~ ) [0%]
# actual   : * user1@localhost.localdomain ( 14K / ~ ) [0%]
# --
# 
```

---

It may not be related to new changes on master and this test just might not be deterministic / stable due to timing now or something. We'll see..

Reply:
We could check the mailbox content, to see where the difference comes from?

Reply:
Now it's back to 13k, so something is not stable there:

```
not ok 318 checking accounts: listmailuser (quotas enabled) in 276ms
# (from function `assert_output' in file test/test_helper/bats-assert/src/assert.bash, line 239,
#  in test file test/tests.bats, line 675)
#   `assert_output '* user1@localhost.localdomain ( 14K / ~ ) [0%]'' failed
# 
# -- output differs --
# expected : * user1@localhost.localdomain ( 14K / ~ ) [0%]
# actual   : * user1@localhost.localdomain ( 13K / ~ ) [0%]
# --
# 
```

Reply:
This was my suspicion as well - sometimes it's 13KB and sometimes 14KB. What I saw locally was that the update check was running sometimes, and sometimes it was too slow: that's basically a race condition. We could therefore try to disable the update checks in the tests and revert to 13KB to see whether this is solved.

Reply:
@georglauterbach, set it like this?

```sh
run docker exec mail /bin/sh -c "sed -i '/ENABLE_QUOTAS=0/d' /etc/dms-settings; echo 'ENABLE_UPDATE_CHECK=0' >> /etc/dms-settings; listmailuser | head -n 1"
```

Reply:
@craue I think it was suggested to set the container ENV at the start of the test file like this one:

https://github.com/docker-mailserver/docker-mailserver/blob/c6b6f680f5a54b6669df3369c8126ca3a41eafc8/test/tests.bats#L24

Reply:
Alright, @polarathene.
Glad I've asked. 😆
Done. Let's see...

Reply:
So this obviously didn't work. Further suggestions?